element-connector/src/test/java/org/eclipse/californium/elements/runner/TestRepeater.java

```diff
@@ -110,7 +110,7 @@ public void run(final Runner runner, final RunNotifier notifier) {
 		if (0 == maximumRepeats) {
 			LOGGER.info("repeat until error!");
 		} else {
-			LOGGER.info("maximum repeats: {}", maximumRepeats);
+			LOGGER.debug("maximum repeats: {}", maximumRepeats);
```

Comment:
We had a setting for if condition due to logging wrapping. If we consider `else` clause, it's not a logging wrapping case.

Reply:
Sorry, we have two settings about if-statements. The setting about log wrapping should not be the one we need here.